wandb/sdk/artifacts/artifact.py

```diff
@@ -1627,19 +1629,16 @@ def download(
     ) -> FilePathStr:
         """Download the contents of the artifact to the specified root directory.
 
-        Existing files located within `root` are not modified. Explicitly delete `root`
-        before you call `download` if you want the contents of `root` to exactly match
-        the artifact.
+        Existing files located within `root` are not modified. Explicitly delete
+        `root` before you call `download` if you want the contents of `root` to exactly
+        match the artifact.
 
         Arguments:
             root: The directory W&B stores the artifact's files.
             allow_missing_references: If set to `True`, any invalid reference paths
                 will be ignored while downloading referenced files.
-            skip_cache: If set to `True`, the artifact cache will be skipped when
-                downloading and W&B will download each file into the default root or
-                specified download directory.
-            path_prefix: If specified, only files with a path that starts with the given
-                prefix will be downloaded. Uses unix format (forward slashes).
+            skip_cache: If set to `True`, the artifact cache will be skipped when downloading
+                and W&B will download each file into the default root or specified download directory.
```

Comment:
question: curious, is there a reason for dropping the `path_prefix` docstring?  (or if unintentional -- we should probably restore it)

Reply:
@moredatarequired got to it first, but I just realized you might need to `git pull` from main to update with his changes from another recent PR

Reply:
Yeah something weird happened here, the branch got outdated pretty quickly, let me fix this.